castra/core.py

```diff
@@ -17,6 +17,18 @@
 from pandas import msgpack
 
 
+bp_args = bloscpack.BloscpackArgs(offsets=False, checksum='None')
+
+def blosc_args(dt):
```

Comment:
These defaults came about from experimentation on the nyctaxi dataset.  I've found that shuffle works well on integer valued data and very well on categorical or timestamp data but offers relatively slim gains on the floating point values (often lat/long).  

I've also generally found that with decent hard drives the costs of more intense compression quickly overwhelm the space saving benefits, leading me to prefer lower compression settings.